docs/src/env-vars.md

```diff
@@ -1,6 +1,8 @@
 ## Environment Variables
 
-You can also configure Martin using environment variables, but only if the configuration file is not used. See [configuration section](config-file.md) on how to use environment variables with config files. See also [SSL configuration](pg-connections.md#postgresql-ssl-connections) section below.
+You can also configure Martin using environment variables, but only if the configuration file is not used.
```

Comment:
```suggestion
You can configure Martin using environment variables, but only if the configuration file is not used. The configuration file itself may use environment variables if needed.
```

Reply:
may?
I think it can do that or are there cases where we don't?

```suggestion
You can configure Martin using environment variables, but only if the configuration file is not used.
The [configuration file itself can use environment variables](config-file.md) if needed.
```